modules/database/src/ioc/dbStatic/dbLexRoutines.c

```diff
@@ -1119,6 +1119,19 @@ static void dbRecordHead(char *recordType, char *name, int visible)
         return;
     }
 
+    if (recordType[0] == '#' && recordType[1] == 0) {
+        status = dbFindRecord(pdbentry, name);
+        if (status == 0) {
+            dbDeleteRecord(pdbentry);
+            fprintf(stderr, ERL_WARNING ": Record \"%s\" DELETED!\n", name);
+            return; /* done */
```

Comment:
Should the `duplicate` flag be set here as well?  What would (or should) happen with the following?

```
record("#", "foo") {
    field(VAL, "4")
}
```

Reply:
That would result in trouble.
Addressed in [37c1e8b](https://github.com/epics-base/epics-base/pull/505/commits/37c1e8b78ef02954d835e0a3e0cf2c63989bf573).